tests/utils.py

```diff
@@ -0,0 +1,25 @@
+class AbstractTestCase:
```

Comment:
why this class? I don't know man but I see it as OverEngineering because you could do assert directly (simple is better my opinion)

Reply:
It helped me to write test cases faster, because I used to Django Tests format 😂

Reply:
It is not a big deal but adding extra classes every where is not a good idea